### ISSUE TITLE
Autoscale Templates: restrictedSrcAddressApp only allows 0.0.0.0/0

### ISSUE THREAD

**Issue by f5-applebaum** [F5Networks/f5-aws-cloudformation-v2#2]
## Description

BigipExternalSecurityGroup in DAG module needs VPC CIDR added as well so the LB can access VIP.

```
        - IpProtocol: tcp
          FromPort: 80
          ToPort: 80
          CidrIp: !Ref vpcCidr
        - IpProtocol: tcp
          FromPort: 443
          ToPort: 443
          CidrIp: !Ref vpcCidr
```
As a result, 0.0.0.0/0 is only value that can be used for restrictedSrcAddressApp

## Template
https://github.com/F5Networks/f5-aws-cloudformation-v2/blob/main/examples/autoscale/payg/autoscale.yaml

## Severity Level
Severity: 3

**Comment by shyawnkarim** [comment]
We are now tracking this issue internally with ESECLDTPLT-2690.

**Comment by shyawnkarim** [comment]
Closing.

This bug fix was included with [Release 1.1.0.0](https://github.com/F5Networks/f5-aws-cloudformation-v2/releases/tag/v1.1.0.0).
